admin/models/check-form.js

```diff
@@ -37,21 +37,16 @@ const CheckForm = new Schema({
   }
 }, {timestamps: true})
 
-// Make the `id` an incrementing number
-CheckForm.plugin(autoIncrement.plugin, { model: 'CheckForm', startAt: 1 })
-
-CheckForm.pre('save', function (next) {
+CheckForm.pre('save', async function (next) {
   if (!this._id) {
-    throw new Error('Missing _id field')
+    // @TODO: There may be a more efficient way to solve the auto-increment.
```

Comment:
this could easily be the victim of a race condition.  It's quite hard to solve without a table lock.

i suggest we leave the old code as-is until we move to SQL, as i have already solved this by using auto increment for the id field in the checkForm table.

Reply:
OK, I'll leave it as it is for now. I'll update the comment.